node-tests/nodetest-runner.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+var glob = require('glob');
+var Mocha = require('mocha');
+var Promise = require('ember-cli/lib/ext/promise');
```

Comment:
same as in https://github.com/Turbo87/ember-data/blob/master/node-tests/nodetest-runner.js#L5

if this is undesired we should change it in the blueprint test helpers repo